### ISSUE TITLE
About the training data?

### ISSUE THREAD

**Issue by xhb666** [naver-ai/ZIM#5]
![image](https://github.com/user-attachments/assets/43125429-09ab-4488-b499-2c6599d21cf2)

May I ask why you fine-tune using only 1% of the data instead of the entire dataset? Is it because this 1% consists of carefully selected data with better details, or is it chosen randomly? Or is there some other reason?
Looking forward to your reply.
Thanks！

**Comment by qjadud1994** [comment]
The SA1B dataset consists of 1,000 subsets [(here)](https://scontent-ssn1-1.xx.fbcdn.net/m1/v/t6/An8MNcSV8eixKBYJ2kyw6sfPh-J9U4tH2BV7uPzibNa0pu4uHi6fyXdlbADVO4nfvsWpTwR8B0usCARHTz33cBQNrC0kWZsD1MbBWjw.txt?ccb=10-5&oh=00_AYCpXzV05weSfO0mGyI0jRykQYUlWij-F9kdjyZEgGH58g&oe=676271D8&_nc_sid=0fdd51).

Without careful data selection, we used the first 10 subsets (sa_000000-sa_000009).

We are motivated by the fine-tuning strategy in [MobileSAM](https://arxiv.org/pdf/2306.14289), which fine-tunes SAM using 0.1% of the SA1B dataset.

Training ZIM on the full SA1B-Matte dataset may yield better performance, we concluded that fine-tuning with 1% of the SA1B-Matte dataset (containing 2.2M matte labels) was enough to achieve reasonable performance (since ZIM is not trained from scratch).
